src/telcameras_v2/ingress_parser.py

```diff
@@ -12,6 +12,14 @@ def parse_single_message(self, ingress_raw_data):
         data = json.loads(ingress_raw_data)['data']
         observation = data_to_observation(data)
 
+        # Does the sensor exist and is it active
+        store, message = store_data_for_sensor(observation)
+        if not store:
+            # We don't want to store this message, but we don't want to throw an error
+            # For that reason we simply return a mocked model with an id so that the
+            # parser will mark it as parsed succesfully
```

Comment:
Dit snap ik niet. Waarom willen we deze error verbergen in plaats van gewoon terug te geven naar de requester?